cedar-policy/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.2.1] - 2024-05-21
```

Comment:
Probably want this date to be today. Or you might want to leave it as "Coming soon" in case we want to wait for any other bug fixes.

```suggestion
## [3.2.1] - Coming soon
```